documentation/src/main/asciidoc/titles/hibernate/hibernate.asciidoc

```diff
@@ -0,0 +1,23 @@
+//Directory attributes
+:stories: ../../stories
+:topics: ../topics
+:imagesdir: ../{topics}/images
+
+//Community doc attributes
+include::../{topics}/attributes/community-attributes.adoc[]
+
+//Community title attributes
+:toc2:
+:icons: font
+:toclevels: 3
+:numbered:
+:sectanchors:
+:stem: asciimath
+
+//Title
+= Hibernate Second-Level Caching
+
+{brandname} provides an interface so you can use it as a second-level cache (2LC) with Hibernate and store all sessions from applications using the same session factory.
```

Comment:
Not sure about "provides an interface", I'd rather go with just "You can use {brandname} as ..."